icubam/db/store.py

```diff
@@ -755,11 +755,17 @@ def get_visible_bed_counts_for_user(
 
     kargs are passed to get_latest_bed_counts_for_icus() method for additional
     filtering, e.g. max_date.
+
+    Args:
+      user_id: run query for a given user
+      force: run query as admin irrespective of the user_id
     """
     user = self.get_user(user_id)
-    if not user:
-      raise ValueError(f"Cannot find user with id {user_id}")
-    if force or user.is_admin:
+    if force:
+      return self._get_bed_counts_for_icus(None, latest=True, **kargs)
+    elif user is None:
+      raise ValueError("a non null user_id must be provided")
```

Comment:
Maybe use capitals and punctuation while we're at it.

Reply:
Also, maybe `user_id` is not actually null be just the user can't be found, right?